tests/testthat/test_applyChangeMeta.R

```diff
@@ -311,3 +311,22 @@ test_that("sort value labels", {
   expect_equal(out, dfSAV$labels[1:3,])
 })
 
+
+test_that("Multiple meta data conflicts during value recoding", {
+  dat <- data.frame(v1 = factor(c("a", "b", "c", "d", "e")))
+  gads <- import_DF(dat)
+
+  expect_error(recodeGADS(gads, "v1", oldValues = c(1, 2), newValues = c(6, 6), existingMeta = "value_new"),
+               "Duplicated values in 'value_new' causing conflicting meta data in variable v1: 6. Use 'existingMeta' = 'drop' or 'ignore' to drop all related meta data.")
+  expect_error(recodeGADS(gads, "v1", oldValues = c(1, 2, 4), newValues = c(6, 6, 5), existingMeta = "value_new"),
+               "Duplicated values in 'value_new' causing conflicting meta data in variable v1: 6. Use 'existingMeta' = 'drop' or 'ignore' to drop all related meta data.")
+
+  expect_error(recodeGADS(gads, "v1", oldValues = c(1, 2), newValues = c(6, 6), existingMeta = "value"),
+               "Duplicated values in 'value_new' causing conflicting meta data in variable v1: 6. Use 'existingMeta' = 'drop' or 'ignore' to drop all related meta data.")

```

Comment:
Nit: I'm reading this test prior to checking the code. Why does the message mention "value_new", whereas existingMeta = "value", i.e., the messages don't differ? 🤔

Reply:
Mh, this one is tricky for multiple reasons. Besides being options for `exinstingMeta`, `value` and `value_new` are columns in the `changeTable` object (hence the corresponding names), which the error message refers to. 

``` r
library(eatGADS)
a <- getChangeMeta(pisa, level = "value")
head(a)
#>    varName value                                  valLabel missings value_new
#> 2   idstud    NA                                      <NA>     <NA>        NA
#> 3 idschool    NA                                      <NA>     <NA>        NA
#> 4  idclass    NA                                      <NA>     <NA>        NA
#> 5  schtype     1                Gymnasium (academic track)    valid        NA
#> 6  schtype     2                                Realschule    valid        NA
#> 7  schtype     3 schools with several courses of education    valid        NA
#>   valLabel_new missings_new
#> 2           NA           NA
#> 3           NA           NA
#> 4           NA           NA
#> 5           NA           NA
#> 6           NA           NA
#> 7           NA           NA
```

<sup>Created on 2023-09-22 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Nevertheless, this error message is only meaningful if users call `applyChangeMeta()`, not if they call `recodeGADS()` (which then calls `applyChangeMeta()`). I have opened a corresponding issue: #59